machine/completion.md

```diff
@@ -14,12 +14,9 @@ for the bash and zsh shell.
 Make sure bash completion is installed. If you use a current Linux in a non-minimal installation, bash completion should be available.
 On a Mac, install with `brew install bash-completion`
 
-Place the completion scripts in `/etc/bash_completion.d/` (`` `brew --prefix`/etc/bash_completion.d/`` on a Mac), using e.g.
+Place the completion script in `/etc/bash_completion.d/` (`` remove `brew --prefix`/etc/bash_completion.d/`` on a Linux), using e.g.
```

Comment:
I don't understand this change. What is the "remove" word for? Also "on a Linux" is not clear or grammatically correct. Can you please work with @londoncalling on the problem you are trying to address with this PR?